codice/alliance
# CAL-413 Remove configurations policy file

#### What does this PR do?
Removes Alliance specific configurations policy file and inherit DDF file.

#### Who is reviewing it? 
(please choose AT LEAST two reviewers that need to approve the PR before it can get merged; if a component team is listed, at least one of its members needs to approve)
@emmberk @JGatley @clockard

#### Choose 2 committers to review/merge the PR.
(please choose ONLY two committers from below, delete the rest)
@rzwiefel
@stustison

#### How should this be tested?
Configure a CDM in place directory. Ingest a NITF. Verify original and overview derived products can be downloaded and that chipping works.

#### Any background context you want to provide?
This policy file is no longer required after adding privileged actions to UrlResourceReader in https://github.com/codice/ddf/pull/3364.

#### What are the relevant tickets?
[CAL-413](https://codice.atlassian.net/browse/CAL-413)

#### Screenshots (if appropriate)
#### Checklist:
